atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaTest.java

```diff
@@ -252,36 +241,13 @@ public void noCachingByDefault() {
         assertThat(schema.getLockWatches()).isEmpty();
     }
 
-    private void checkIfFilesAreTheSame(List<String> generatedTestTables) {
-        generatedTestTables.forEach(tableName -> {
-            String generatedFilePath =
-                    String.format("com/palantir/atlasdb/table/description/generated/%s.java", tableName);
-            File expectedFile = new File(EXPECTED_FILES_FOLDER_PATH, generatedFilePath);
-            File actualFile = new File(testFolder.getRoot(), generatedFilePath);
-
-            assertThat(expectedFile.length()).isEqualTo(actualFile.length());
-
-            try (Stream<String> expectedFileStream = java.nio.file.Files.lines(expectedFile.toPath());
-                    Stream<String> actualFileStream = java.nio.file.Files.lines(actualFile.toPath());
-                    Stream<Boolean> zipped = correspondingLinesMatchOrAreHashes(expectedFileStream, actualFileStream)) {
-
-                assertThat(zipped).allMatch(elem -> elem);
-            } catch (IOException e) {
-                Assertions.fail("Exception on stream creation", e);
-            }
-        });
-    }
-
-    private static Stream<Boolean> correspondingLinesMatchOrAreHashes(
-            Stream<String> expectedFileStream, Stream<String> actualFileStream) {
-        return Streams.zip(
-                expectedFileStream,
-                actualFileStream,
-                (first, second) -> first.equals(second) || (first.contains(CLASS_HASH) && second.contains(CLASS_HASH)));
-    }
+    private void checkIfFilesAreTheSame(String generatedFileName) {
+        String generatedFilePath =
+                String.format("com/palantir/atlasdb/table/description/generated/%s.java", generatedFileName);
+        File expectedFile = new File(EXPECTED_FILES_FOLDER_PATH, generatedFilePath);
+        File actualFile = new File(testFolder.getRoot(), generatedFilePath);
 
-    private String readFileIntoString(File baseDir, String path) throws IOException {
-        return new String(Files.toByteArray(new File(baseDir, path)), StandardCharsets.UTF_8);
+        assertThat(expectedFile).hasSameTextualContentAs(actualFile);
```

Comment:
Simplified tests to use AssertJ's `AbstractFileAssert` instead of our hand-rolled line-based comparison.

Reply:
nice, thanks! yeah that makes sense :)